plugins/source_test.go

```diff
@@ -237,19 +260,26 @@ func testSyncTable(t *testing.T, tc syncTestCase) {
 			resources)
 	})
 
-	var i uint64
-	for range resources {
+	var i int
+	for resource := range resources {
+		if tc.data == nil {
+			t.Fatalf("Unexpected resource %v", resource)
+		}
+		if i > len(tc.data) {
+			t.Fatalf("expected %d resources. got %d", len(tc.data), i)
+		}
+		// if !tc.data[i].Equal(resource.GetValues()) {
+		// 	t.Fatalf("expected in item %d %v. got %v", i, tc.data[i], resource.GetValues())
+		// }
```

Comment:
```suggestion
		// if !tc.data[i].Equal(resource.GetValues()) {
		// 	t.Fatalf("expected in item %d %v. got %v", i, tc.data[i], resource.GetValues())
		// }
```

Reply:
Was this an accidental suggestion @erezrokah, or is there something different between the two versions that I'm not seeing? 👓

Reply:
Ah, I meant to remove those if not used, good catch @hermanschaaf

Reply:
added back